src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -64,6 +65,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
+            ++m_version;
```

Comment:
> What if CompareInfo in netfx suddenly uses the field for something else and (obviously) expects it to contain the correct value?

Then that would be introducing a behavioral difference, which would a) break compatibility with existing versions, and b) be something we'd need to do in corefx as well.